.github/ISSUE_TEMPLATE/cran-release.yaml

```diff
@@ -0,0 +1,124 @@
+---
+name: 🎉 CRAN Release
+description: Template for release to CRAN
+title: "[Release]: <version>"
+labels: ["release"]
+assignees:
+  - KlaudiaBB
+  - cicdguy
+  - shajoezhu
```

Comment:
@donyunardi instead?

Reply:
or package maintainer?

Reply:
for all the packages?

Reply:
I honestly don't know - up to you. Please discuss it during the daily / slack

Reply:
This .yaml file is propagated from either insightsengineering/.github or insightsengineering/.github-private. If we make changes here, they will still be overwritten during the next propagation. If modifications are necessary, they should be made in those repos.

Reply:
Valid point. Let's leave it then and I will create a separate issue to agree on this and propagate

Reply:
https://github.com/insightsengineering/idr-tasks/issues/722